integration-tests/tests/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "es2018",
+		"target": "es2020",
```

Comment:
I remember there was a strange TS issue where the integration tests build output was not useable by one of the environments.  Might have to revert this and document what that was.